### PR TITLE
bash: version bump to 4.4.12

### PR DESCRIPTION
**Pull request by hashashin**
Signed-off-by: Iván Atienza <gentoo.power@gmail.com>

Maintainer: @Naoir 
Compile tested: mips, lede master, lede v17.01.2
Run tested: mips, router model HG553, ~2 weeks normal usage lede v17.01.2

Description:
Bash 4.4 with all upstream patches until date .12 (adapted via quilt). Refreshed openwrt specific patches when needed.

### CODE REVIEW

**Comment by champtar** [comment]
Hi @hashashin 
I've pushed a conflicting commit, can you rebase?

File: utils/bash/Makefile [hunk 1]
```diff
@@ -1,21 +1,21 @@
 #
-# Copyright (C) 2007-2015 OpenWrt.org
+# Copyright (C) 2007-2017 OpenWrt.org
```
Comment (champtar) [inline utils/bash/Makefile]: please don't update OpenWrt copyright

**Comment by champtar** [comment]
ping @Naoir

**Comment by diizzyy** [comment]
As far as I can tell this runs just fine on my Linksys WRT3200ACM.
I think we can push this due to maintainer timeout.

**Comment by diizzyy** [comment]
@champtar @hnyman 
Can we push this due to maintainter timeout?

**Pull request merged**